config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -694,3 +694,52 @@ periodics:
           cpu: 2
   rerun_auth_config:
     allow_anyone: true
+- interval: 24h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-e2e-kind-arm64
+  annotations:
+    testgrid-dashboards: sig-testing-kind
+    testgrid-tab-name: kind-master-arm64
+    description: Uses kubetest to run e2e tests against a latest kubernetes master created with sigs.k8s.io/kind using arm64
+    testgrid-alert-email: bentheelder@google.com,antonio.ojea.garcia@gmail.com, davanum@gmail.com
+    testgrid-num-columns-recent: '6'
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decorate: true
+  decoration_config:
+    timeout: 60m
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: us-central1-docker.pkg.dev/k8s-staging-test-infra/images/kubekins-e2e:v20240705-131cd74733-master
```

Comment:
@BenTheElder this image can replace the krte one in the future.

Reply:
@upodroid do we need a different name? `kubekins-e2e` is overloaded here? the images are different in the existing and this new location no?

Reply:
(well! not for this PR, but just an observation)

Reply:
> @BenTheElder this image can replace the krte one in the future.

I'm hesitant to do that, we made KRTE specifically to avoid the blast radius of kubekins changes affecting kind jobs.

It also gives room to iterate on things like dind without it affecting _every_ CI job.